diff -c --recursive a/Makefile b/Makefile
*** a/Makefile	2012-10-01 05:29:41.000000000 +1000
--- b/Makefile	2020-03-01 18:32:47.538708197 +1000
***************
*** 3,9 ****
  #------------ Debug value -------------
  DEBUG=no
  #------------ Debian values -----------
! DESTDIR=
  VERSION=1.15
  NODEPENDS=
  PKGDIR=$(DESTDIR)/usr/share/algol68toc
--- 3,9 ----
  #------------ Debug value -------------
  DEBUG=no
  #------------ Debian values -----------
! DESTDIR=${PREFIX}
  VERSION=1.15
  NODEPENDS=
  PKGDIR=$(DESTDIR)/usr/share/algol68toc
***************
*** 55,61 ****
  #------------ Programs -----------
  ALGOL=$(CTD)/a68toc
  SHELL=/bin/sh
! INSTALL=$(shell which install) -g root -o root
  INSTALLDATA=$(INSTALL) -m 644
  INSTALLPROG=$(INSTALL) -m 555
  
--- 55,61 ----
  #------------ Programs -----------
  ALGOL=$(CTD)/a68toc
  SHELL=/bin/sh
! INSTALL=$(shell which install)
  INSTALLDATA=$(INSTALL) -m 644
  INSTALLPROG=$(INSTALL) -m 555
  
